content/articles/matlab-graphical-user-interface/index.md

```diff
@@ -0,0 +1,104 @@
+### GRAPHICAL USER INTERFACE IN MATLAB
+## Introduction
```

Comment:
Please start with a proper introduction.

Talk about what Matlab is, what GUI is, how GUI can be built using Matlab. Then, mention what this article is about, what are the key takeaways.